impeller/renderer/backend/vulkan/render_pass_builder_vk.h

```diff
@@ -54,10 +54,19 @@ class RenderPassBuilderVK {
   // Visible for testing.
   const std::optional<vk::AttachmentDescription>& GetDepthStencil() const;
 
+  // Visible for testing.
+  std::optional<vk::AttachmentDescription> GetColor0() const;
+
+  // Visible for testing.
+  std::optional<vk::AttachmentDescription> GetColor0Resolve() const;
```

Comment:
Please make the test a friend instead of making these public (example https://github.com/flutter/engine/blob/c7f553eb6a3083d0b66621b16caccb3b0cc3a384/impeller/renderer/backend/gles/buffer_bindings_gles.h#L51)

Reply:
Listing out each test name seems like a pretty big step down, given that this is an internal library of an internal library.